code/game/machinery/cloning.dm

```diff
@@ -219,6 +219,11 @@
 			var/datum/quirk/Q = new V(H)
 			Q.on_clone(quirks[V])
 
+		for(var/t in traumas)
+			var/datum/brain_trauma/BT = t
+			var/datum/brain_trauma/cloned_trauma = new BT.type(BT.on_clone())
+			H.gain_trauma(cloned_trauma, BT.resilience)

```

Comment:
I think it's better to have the on_clone() proc be called from the old trauma take the new trauma as an argument and let it do anything necessary, since not all traumas have initialize arguments that suit this purpose.